app/views/standup_meeting_groups/show.html.erb

```diff
@@ -1,4 +1,5 @@
-<%= render @standup_meeting_group %>
-<div>
-  <%= link_to "Back to standup meeting group", standup_meeting_groups_path %>
-</div>
+<%= render partial: @standup_meeting_group, locals: { 
```

Comment:
not sure whether we actually need to have this in a partial, but 🤷

Reply:
prob not but 🤷